.github/workflows/ci.yaml

```diff
@@ -88,7 +88,9 @@ jobs:
           tags: smartonfhir/cumulus-etl:latest
 
       - name: Download NLP images
-        run: docker compose --profile covid-symptom up -d --quiet-pull
+        run: |
+          # Just start the specific services we need, to conserve CI disk space
+          docker compose up -d --quiet-pull cnlpt-negation ctakes-covid
```

Comment:
Unrelated fix put in as a separate commit - CI was failing because the image was running out of space... GitHub gives us 14GB (but then of course the OS and other stuff that actions install like node etc take up a lot).